src/hotspot/share/classfile/stackMapTable.cpp

```diff
@@ -143,10 +137,14 @@ void StackMapTable::print_on(outputStream* str) const {
   str->print_cr(" }");
 }
 
-StackMapReader::StackMapReader(ClassVerifier* v, StackMapStream* stream, char* code_data,
-                               int32_t code_len, TRAPS) :
-                               _verifier(v), _stream(stream),
-                               _code_data(code_data), _code_length(code_len) {
+StackMapReader::StackMapReader(ClassVerifier* v, StackMapStream* stream,
+                               char* code_data, int32_t code_len,
+                               StackMapFrame* init_frame,
+                               u2 max_locals, u2 max_stack, TRAPS) :
+                               _verifier(v), _stream(stream), _code_data(code_data),
```

Comment:
Could you indent these out 4 so they don't look like another 4 lines of parameters?